c_examples/regs-dump.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <string.h>
+#include "libmicrovmi.h"
+
+void read_registers(MicrovmiContext* driver, const char* vm_name) {
+    if (microvmi_pause(driver) == MicrovmiSuccess) {
+        printf("Paused.\n");
+    } else {
+        printf("Unable to pause VM.\n");
+        return;
+    }
+    Registers regs;
+    memset(&regs, 0, sizeof(regs));
+    if (microvmi_read_registers(driver, 0, &regs) == MicrovmiSuccess) {
+        printf("rax: 0x%llx\n", regs.x86._0.rax);
+        printf("rbx: 0x%llx\n", regs.x86._0.rbx);
+        printf("rcx: 0x%llx\n", regs.x86._0.rcx);
+        printf("rdx: 0x%llx\n", regs.x86._0.rdx);
```

Comment:
I believe we should also print `RIP`, `RSP`, `RBP`, `RSI`, `RDI` and the `RFLAGS` to show meaningful output (for 32 bits systems at least)